src/Tests/NamingIntegrationTests.cs

```diff
@@ -1,41 +1,41 @@
-﻿using StatsdClient.Configuration;
-using NUnit.Framework;
+﻿//using StatsdClient.Configuration;
+//using NUnit.Framework;
 
-namespace Tests
-{
-    [TestFixture]
-    public class NamingIntegrationTests
-    {
-        [Test]
-        public void environment_name()
-        {
-            Assert.That(Naming.CurrentEnvironment,Is.EqualTo("environment"));
-        }
+//namespace Tests

```

Comment:
Was actually planning to remove this in a future version as it's been Obsolete for a while - seems as good a time as any if you wouldn't mind just deleting the tests too.